SpatialGDK/SpatialGDK.uplugin

```diff
@@ -1,7 +1,7 @@
 {
   "FileVersion": 3,
-  "Version": 3,
-  "VersionName": "0.6.0",
+  "Version": 4,
```

Comment:
Hmm, 0.7.0 is version 4. Ideally we have a separate mapping of version names to version numbers, and ensure non are reused.

Reply:
We could potentially also not increment the `version` when doing hotfixes. Just for major releases.